docs/operator_context.md

```diff
@@ -0,0 +1,112 @@
+# Operator context
+
+This is a resource used to define values for communicating with admin api of
+tyk ce/pro deployment.
+
+`OperatorContext` can be referenced on all custom resources by `contextRef` property.
+When a custom resource is applied with `contextRef` set then all the operations
+conducted by the operator will use the `OperatorContext` supplied by the `contextRef` 
+to perform reconciliation.
+
+# Defining OperatorContext
+
+Annotated `OperatorContext` for a community edition deployment looks like this
+
+```yaml
+apiVersion: tyk.tyk.io/v1alpha1
+kind: OperatorContext
+metadata:
+  name: community-edition
+spec:
+  env:
+    # The mod e of the admin api
+    # ce - community edition (open source gateway)
+    # pro - dashboard (requires a license)
+    mode: ce
+    # The authorization token this will be set in x-tyk-authorization header on the
+    # client while talking to the admin api
```

Comment:
This is true for CE only - for Pro, or Cloud control-planes - this is the access token for the dashboard API.

See this table for reference terminology:

https://github.com/TykTechnologies/tyk-operator/blob/master/docs/installation/installation.md#tyk-operator-configuration

Reply:
check the line above the code block 
```
Annotated `OperatorContext` for a community edition deployment looks like this
```

The comment is for community edition